addons/breathing/functions/fnc_handleBreathing.sqf

```diff
@@ -64,6 +64,25 @@ if (!local _unit) then {
     private _multiplierOxygen = GVAR(BVMOxygen_Multiplier);
     private _perfusionActive = false;
 
+    if (GVAR(SpO2_cardiacActive)) then {
+        private _ht = _unit getVariable [QEGVAR(circulation,ht), []];
+
+        if (GVAR(SpO2_cardiacValue) >= _status) then {
```

Comment:
shouldnt that be lesser than? doesnt make sense to "cause" hypoxia above set value
```suggestion
        if (GVAR(SpO2_cardiacValue) =< _status) then {
```

Reply:
I would like to revert this logic because its more readable that way imo
```suggestion
        if (_status <= GVAR(SpO2_cardiacValue)) then {
```